packages/link/Link.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import isAbsoluteUrl from 'is-absolute-url';
+
+// if absolute, return url. otherwise loadappify the url
+export const getUrl = (url = '') => {
+  const absolute = isAbsoluteUrl(url);
+  if (absolute) return url;
+
+  return `/public/apps/home/#!/loadApp?appUrl=${encodeURIComponent(url)}`;
+};
+
+const AvLink = ({ tag: Tag, url, target, children, ...props }) => (
+  <Tag href={getUrl(url)} target={target} data-testid="av-link-tag" {...props}>
+    {children}
+  </Tag>
+);
+
+AvLink.defaultProps = {
+  tag: 'a',
+};
+
+AvLink.propTypes = {
+  url: PropTypes.string.isRequired,
```

Comment:
Not sure if we should or not here. But maybe still using the same default name of `href` as the prop name since it is in fact an href of the tag.

Also, I know this is bad practice but if this `Tag` ever needs to be anything other than an anchor tag say a `button`, we should expose the url as well in the `onClick` property possibly?

```javascript
// sudo code. But prop on onClick can accept the url and the event
({ onClick}) =>
onClick={(event) => onClick(get(url),event)}
```

Reply:
yeah - i'm fine either way calling this prop `url` or `href`. i went with `url` because what you pass in as the prop isn't necessarily what gets rendered as the `href`.  ¯\_(ツ)_/¯

i'll add the `onClick` functionality 👍

Reply:
Its potato or potato, didn't know your thoughts on that.  We can keep as `url`

Reply:
8e3a646769d78b4ba4c6b599636bf5e2b15bda19